docs/api/Shell/addCommand.md

```diff
@@ -5,11 +5,13 @@ Adds a command to the end of the pipeline of the shell object.
 #### Arguments
 
 * **command** - A PowerShell command or a path to a PowerShell script `[String] _required_`
-* **params** - A Set of parameters to be added to the command. `[Array/String] (Default: []) _optional_`
+* **params [DEPRECATED]** - A Set of parameters to be added to the command. `[Array/String] (Default: []) _optional_`
   * {name: '', value: ''}
   * {name: value}
   * 'switch'
 
+> addCommand() with the @param syntax is no longer supported. please use ES6 string templates instead.
+
 #### Returns
 
 [Promise] - A promise that resolves with the array of commands currently in the pipeline, or rejects with an error.    
```

Comment:
resolves -> fulfills

Reply:
resolve -> when the promise's fate is determined by either getting a value, an error (rejection) or when it is set to "follow" another promise.

fulfill -> when the promise gets a value.